### PR TITLE
Version 0.5.0 - Adding support for .NET Standard 2.0

### PR DESCRIPTION
**Pull request by kristofdc**
Version 0.5.0 contains following changes:

**Build automation**
- Adding appveyor.yml file to automate Nuget build, test and pack process

**Enhancements**
- Moving sources into src folder
- Adding support for .NET Framework 4.5.2/4.6.1 & .NET Standard 2.0

**Breaking changes**
- Namespace changed to Castle.Facilities.Quartz, (code remains the same)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  only:
+  - master
+  
+configuration: Release
+
+platform: Any CPU
+
+init:
+  # Good practise, because Windows line endings are different from Unix/Linux ones
+  - cmd: git config --global core.autocrlf true
```
Comment (jonorossi) [inline appveyor.yml]: There should be no need to modify `core.autocrlf` from its default on any platform. I've fixed the line ending issues by adding a .gitattributes file and normalizing all existing files in the object database (#22).

Could you please rebase your pull request to include those changes as your pull request already has some line ending changes.

**Comment by kristofdc** [comment]
Rebase was not evident, because the whole folder structure changed and some files were moved.
.gitattributes added

**Comment by jonorossi** [comment]
@kristofdc problem is this pull request has conflicts attempting to merge to master, see the GitHub status section.

**Comment by jonorossi** [comment]
@kristofdc if rebase gives you issues and you are running Windows, the easiest way to resolve this is to branch off upstream's master which now has a .gitattributes file, on Windows the working copy before and after this change had CRLFs, delete all files (except .git/ and .gitattributes) from the file system and copy your moved and modified files (which also have CRLFs on Windows).

**Comment by kristofdc** [comment]
@jonorossi I do not succeed to rebase it without conflicting files in the pull request. It seems that all conflicts are not existing files... except the .gitattributes and the License.txt I suppose.
I need some more help or you need to delete those files while merging into master.

**Comment by jonorossi** [comment]
> I do not succeed to rebase it without conflicting files in the pull request. It seems that all conflicts are not existing files... except the .gitattributes and the License.txt I suppose.
> I need some more help or you need to delete those files while merging into master.

I thought you'd hit that problem with rebase, did you try the suggestion in my last comment? Basically make a copy of the files, reset your branch to origin/master, delete the files (except 2) from the file system/working copy (no git operations), copy your modified files into the empty directory, stage and commit the changes, then force push your branch?

**Comment by kristofdc** [comment]
@jonorossi  I was missing the git reset in the whole story -> now you are able to merge

**Comment by jonorossi** [comment]
Even GitHub won't let me "squash and merge" your 44 commits with "+2,869 −3,469" changes.

Rather than more back and forth I've pulled your branch and squished it locally, it is committed as 
7f9f3c6f19830a78ebb127ddccc824c345b726ca on the master branch with you as the author.

With that I'll close this pull request as the changes are merged. Thanks heaps. I'll get this repository set up on the castleproject AppVeyor account.

**Pull request closed without merging**